src/beacon-api/app/services/scheduler_service.py

```
@@ -202,7 +202,7 @@ async def compute_and_store_performance(
             return {
                 "success": True,
                 "skipped": True,
-                "message": "All requested days already have complete data",
+                "message": f"All requested days already have complete data",
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)